python/lsst/meas/base/__init__.py

```diff
@@ -46,3 +46,4 @@
 from .transforms import *
 from .wrappers import *
 from .compensatedGaussian import *
+from .colorUtilities import *
```

Comment:
It looks like these are arranged alphabetically, except for `.compensatedGaussian`. I would the new line in alphabetical order, and fix `.compensatedGaussian` too.